pandas/conftest.py

```diff
@@ -60,6 +60,23 @@ def spmatrix(request):
     return getattr(sparse, request.param + '_matrix')
 
 
+@pytest.fixture(params=[0, 1, 'index', 'columns'],
+                ids=lambda x: "axis {!r}".format(x))
+def axis(request):
```

Comment:
maybe also add axis_frame just for consistency? (its exactly equal to axis)

Reply:
sure, ok